src/main/java/com/example/librarymanagementsystem/model/Patron.java

```diff
@@ -0,0 +1,69 @@
+package com.example.librarymanagementsystem.model;
+
+public class Patron {
+
+    private int userId;
+    private String username;
+    private String email;
+    private String password;
+
+    /**
+     * Instantiates a new Patron.
+     *
+     * @param username the username
+     * @param email    the email
+     * @param password the password
+     */
+    public Patron(String username, String email, String password) {
+        this.username = username;
+        this.email = email;
+        this.password = password;
+    }
+
+    /**
+     * Instantiates a new Patron.
+     *
+     * @param userId   the user id
+     * @param username the username
+     * @param email    the email
+     * @param password the password
+     */
+    public Patron(int userId, String username, String email, String password) {
+        this.userId = userId;
+        this.username = username;
+        this.email = email;
+        this.password = password;
+    }
```

Comment:
Instead of defining strict constructors for other parameters, use builders (`Builder Design Pattern`) to make it more flexible, readable and reducing error encounters. Look into that and see how it works.